src/erp/reports/ReportGenerator.java

```diff
@@ -0,0 +1,33 @@
+package erp.reports;
+
+import erp.sales.SalesManager;
+import erp.payment.PaymentProcessor;
+import erp.employee.EmployeeManager;
+import java.util.List;
+
+public class ReportGenerator {
+
+    public void generateMonthlySalesReport() {
+        SalesManager salesManager = new SalesManager();
+        PaymentProcessor paymentProcessor = new PaymentProcessor();
+        EmployeeManager employeeManager = new EmployeeManager();
+
+        List<String> salesData = salesManager.getMonthlySalesData();  
+        List<String> paymentData = paymentProcessor.getPaymentData();
+        List<String> employeeData = employeeManager.getEmployeeCommissionData();
+
+        System.out.println("Generating Monthly Sales Report...");
+
+        for (String sale : salesData) {
+            System.out.println("Sale: " + sale);
```

Comment:
Printing the report might not be the best approach. Consider creating a text file or CSV file for easier viewing.